website/docs/docs/collaborate/project-recommendations.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Project recommendations"
+sidebar_label: "Project recommendations"
+description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
+---
+
+:::tip Public preview 
```

Comment:
Should these be a snippet to ensure consistency on different pages and in future cahnges?